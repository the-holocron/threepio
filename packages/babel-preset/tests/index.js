const config = require('../babel.config.json');

describe('Preset', () => {
	describe('Babel', () => {
		it('should render the configuration', () => expect(config).toMatchSnapshot());
	});
});
