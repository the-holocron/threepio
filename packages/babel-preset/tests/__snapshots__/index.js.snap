// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preset Babel should render the configuration 1`] = `
Object {
  "plugins": Array [
    "@babel/plugin-proposal-class-properties",
    Array [
      "@babel/plugin-proposal-object-rest-spread",
      Object {
        "useBuiltIns": true,
      },
    ],
  ],
  "presets": Array [
    Array [
      "@babel/preset-env",
    ],
  ],
}
`;
